esrally/driver/__init__.py

```diff
@@ -16,4 +16,11 @@
 # under the License.
 
 # expose only the minimum API
-from .driver import DriverActor, PrepareBenchmark, PreparationComplete, StartBenchmark, BenchmarkComplete, TaskFinished
+from .driver import (
```

Comment:
It seems that it is configurable how multi-line imports are rendered. Can we please use `11 - Backslash Grid` for such cases (see their [docs](https://pycqa.github.io/isort/docs/configuration/multi_line_output_modes/))?

Reply:
While black does not *sort* imports, it does reformat them: we can only use the black mode in isort, not the other ones. black does this to minimize diffs.